Extension/src/packageManager.ts

```diff
@@ -384,7 +384,7 @@ export class PackageManager {
                             zipfile.readEntry();
                         });
                     } else {
-                        util.checkFileExists(absoluteEntryPath).then((exists: boolean) => {
+                        util.checkDirectoryExists(absoluteEntryPath).then((exists: boolean) => {
```

Comment:
Isn't this a file and not a directory? The "if" part above appears to handle directories.

Reply:
I am not sure about this one.